src/main/java/org/jabref/logic/importer/fetcher/INSPIREFetcher.java

```diff
@@ -59,33 +56,38 @@ public String getName() {
     @Override
     public URL getURLForQuery(String query) throws URISyntaxException, MalformedURLException, FetcherException {
         URIBuilder uriBuilder = new URIBuilder(INSPIRE_HOST);
-        uriBuilder.addParameter("p", query); // Query
-        //uriBuilder.addParameter("jrec", "1"); // Start index (not needed at the moment)
-        uriBuilder.addParameter("rg", "100"); // Should return up to 100 items (instead of default 25)
-        uriBuilder.addParameter("of", "hx"); // BibTeX format
+        uriBuilder.addParameter("q", query); // Query
```

Comment:
there is no paging support (start + size) for the new interface?

Reply:
The json api has support for it, but when calling with the bibtex header not.

Reply:
On my end `&size=10&page=2` seems to works fine as of today, not sure about a week ago.

Reply:
Indeed, this seems to work. I will create a follow up PR